ansible/environments/local/group_vars/all

```diff
@@ -59,6 +59,11 @@ runtimes_manifest:
       image:
         name: "action-swift-v4.2"
       deprecated: false
+    - kind: "swift:5.1"
+      default: true
```

Comment:
```suggestion
      default: false
```

Reply:
Good spot. What's the meaning of default? Do we want to leave `"swift:4.2"` as default?

Reply:
Sure any, just one needs to be default. 
But first check that there are no tests that assume 3.1 is the default